charts/cloudnative-pg/Chart.yaml

```diff
@@ -15,7 +15,7 @@
 #
 apiVersion: v2
 name: cloudnative-pg
-description: CloudNativePG Helm Chart
+description: CloudNativePG Operator Helm Chart
```

Comment:
```suggestion
description: CloudNativePG ClusterHelm Chart
```

Reply:
Uhmm.. That's the operator chart: `charts/cloudnative-pg/Chart.yaml`, as opposed to ``charts/cluster/Chart.yaml`. The suggestion doesn't make sense.

Reply:
Ohh shoot, misread sorry!